app/src/main/java/com/example/mycleaninglog/RoomSettingsDialogBox.kt

```diff
@@ -143,6 +143,8 @@ fun RoomSettingsDialogBox(c: Context, selectedRoomSettings: myRoom, viewModel: M
                     //button for adding a task
                     Button(
                         onClick = {
+                            //saveCleaningTask(addTask.toString(), "Custom", viewModel, selectedRoomSettings)
```

Comment:
Avoid pushing commented code to Version Control.  Version control has tools that makes this unnecessary.
- You can look at history to see what used to be there.
- You can use branches for experimental features.